ai/smart-labeling/quickstart.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Smart Labeling Quickstart
+description: This page shows you how to get started with Scaleway Elements Bare Metal Servers.
+dates:
+  validation: 2021-11-04
+  posted: 2021-11-04
+---
+
+Scaleway [Smart Labeling)(ADD_LINK) is a plattform that allows you to annnotate images or videos (whether by yourself or via a team of annotators). The annotation interface that Smart Labeling is currently using is CVAT: the [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat) developed by Intel. Smart Labeling facilitates using CVAT in the cloud-native manner, with labeling software running on-demand, annotations safely persisted in an automatically backed up database, and datasets stored in cost-efficient S3 buckets.
+
+Our solution comes with everything needed to start annotating: whether on your own or with the help of others (referred to as annotators). Annotators do not need to create Scaleway accounts, thy can log in directly to the CVAT annotation interface, whose login page is publicly accessible.
+
```

Comment:
For rest, apply suggestions from the how tos to the quick start bits copied/pasted too